Swiftfin tvOS/Views/UserSignInView.swift

```diff
@@ -146,11 +140,48 @@ struct UserSignInView: View {
             }
             .padding(.vertical)
 
-            Text(viewModel.quickConnectCode ?? "------")
+            Text(quickConnectCode)
```

Comment:
Let's instead make it so that the steps are always on screen and the code area shows the loading/error/code state.